### ISSUE TITLE
the final representation of the code 

### ISSUE THREAD

**Issue by Lydiatingting** [Jumpat/SegAnyGAussians#118]
Hello! I want to ask, will the ply file be generated after the "python saga_gui.py --model_path <path to the pre-trained 3DGS model>" step? I also want to ask, what is the final representation of the code after it has run? “python render.py -m <path to the pre-trained 3DGS model> --precomputed_mask <path to the segmentation results> --target  scene --segment
"Will the code re-produce the point cloud after running, why do I only have pictures?"

**Comment by Jumpat** [comment]
Yeah. Rendering the results can only get images. In the SAGA GUI, it generate a 0-1 bitmap, which depict the Gaussians belong to the target of interest. The point cloud (3D Gaussians) is still stored in the original 3D-GS model. You can use the bitmap to cut off the original Gaussian model to get the target.
